dashboard/src/reducers.ts

```diff
@@ -0,0 +1,181 @@
+import * as _ from 'lodash';
+import { combineReducers } from 'redux';
+
+import { Action } from './actions';
+import * as conv from './converters';
+import {
+    AuthenticationState, ParkingRegionMapState, RegionsMap,
+    RegionUsageHistory, RootState, ViewState } from './types';
+
+// Auth state reducer ////////////////////////////////////////////////
+
+const initialAuthState: AuthenticationState = {
+    loggedIn: false,
+    existingLoginChecked: false,
+};
+
+function auth(
+    state: AuthenticationState = initialAuthState,
+    action: Action,
+): AuthenticationState {
+    if (action.type === 'CHECK_EXISTING_LOGIN') {
```

Comment:
As was commented above, concider using constants.

Reply:
That would only add more code without actually improving anything.

Tried to change that string on the line to `'CHECK_EXISTING_LOGIN2'` and this is what TypeScript says after that:
```
/home/tuomas/projects/hel/parkkihubi/app/dashboard/src/reducers.ts
(21,9): Operator '===' cannot be applied to types '"CHECK_EXISTING_LOGIN" | "RESOLVE_EXISTING_LOGIN_CHECK" | "REQUEST_CODE_TOKEN" | "RECEIVE_CODE_TO...' and '"CHECK_EXISTING_LOGIN2"'.
```

Reply:
Ok